resources/views/user/includes/header.blade.php

```diff
@@ -20,7 +20,7 @@
                 </li>
 
                 <li class="nav-item px-lg-2">
-                    <a class="nav-link" href="#">Applications</a>
+                    <a class="nav-link" href="{{route('user.jobs' , auth()->id() )}}">Applications</a>
```

Comment:
Why you send the current user id ??!
Are you use this route in another view?

Reply:
This is the user's applications view so i send user id !!